### ISSUE TITLE
Banned word

### ISSUE THREAD

**Issue by JDZC** [gunthercox/ChatterBot#1823]
Hi I have another question! (Sorry for bother, I'm using chatterbot for a school project)

How did I exactly use the excluded words parameter? I can't get any progress trying to use it and I need a dirty word filter. 
Also, the paramater allows me to anserw something back?
Like: 
User: fuck (excluded word) 
Bot: This is not a good thing to say

**Comment by vkosuri** [comment]
Yes it is possible, you have to write your own pre processer to block such words https://chatterbot.readthedocs.io/en/stable/preprocessors.html

**Comment by ChristianTremblay** [comment]
@vkosuri Is using preprocessor better than a logical Adapter for speed reasons in the context of dealing with that kind of input ? Or is there another reason ?
